cryptoxide/README.md

```diff
@@ -0,0 +1,79 @@
+# Cryptoxide
+
+[![Build Status](https://travis-ci.org/input-output-hk/rust-cardano.svg?branch=master)](https://travis-ci.org/input-output-hk/rust-cardano)
+![MIT or APACHE-2 licensed](https://img.shields.io/badge/licensed-MIT%20or%20APACHE--2-blue.svg)
+
+A pure-rust implementation of various modern cryptographic algorithms, which no dependencies
+and no foreign code (specially C or assembly), to work with WASM and embedded devices.
+
+This crates is aiming to support as many architectures as possible with as
+little dependencies as possible.
```

Comment:
This crates package aims to support as many architectures as possible with as
little dependencies as possible.